rundeck-agent/src/main/kotlin/com/hadihariri/teamcity/plugins/rundeck/agent/RunDeck.kt

```diff
@@ -63,7 +64,11 @@ public object RunDeck {
         if (execution.code == 200) {
             println(ServiceMessage.asString("rundeck", mapOf("text" to "Job ${rundeckOptions.jobId} launched successfully with id ${execution.result}", "status" to "NORMAL")))
             if (rundeckOptions.waitFinish) {
-                while (counter < 1200) {
+                var timeout: Long = 2700
```

Comment:
What timeout feature in rundeck does this control?

Reply:
Its the timeout for teamcity waiting on the rundeck job to finish